templates/volontulo/statics/site-news.html

```diff
@@ -0,0 +1,16 @@
+{% load bootstrap3 %}
+
+{% bootstrap_css %}
+
+<div class="media-body">
+    <div class="media-left">
+        21 października 2015
+    </div>
+    <div class="media-body">
+        <h3 class="media-heading">News o działaniu portalu</h3>
```

Comment:
1. Yes, but it shouldn't be `<h1>` since it is usually the page name
2. `<h2>` could be 'Page news' if we gonna have multiple entries 

Since we don't have 1) and we don't know 2) it is impossible to decide which heading we should use. We will know that later.